### ISSUE TITLE
Configuration Problem Dyson Cloud / Dyson Local

### ISSUE THREAD

**Issue by pelu89** [shenxn/ha-dyson#96]
Hey, 

I am fairly new to Home Assistant. I tried configure the Dyson integrations but I cannot login with my Dyson account. When I enter my mail address I cannot go further because I get an error "Unknown error occurred". When I try to enter via the Dyson local integration, I put in all the required information but then I get an error "Failed to connect". I have not yet tried the "Wifi Setup" as I am not quite sure where to find the wifi information for my device.

Is there any help you can offer concerning these specific problems? 

Greetings, 

Peter

**Comment by blairscarlett** [comment]
I was able to get this to work by using Cloud Local. Grab the Wifi SSID and password from the device. If you removed the sticker, it is behind the filter. Put the Dyson into wifi pairing mode. Start by adding the device in HA using Dycon local and select Wifi, enter you Wifi SSID and password but dont hit confirm. Then switch over to the Dyson app on your phone and start to add fan to the Dyson app. After you have connected the phone to the Wifi of the Dyson and the app is transferring the Wifi settings, you can click confirm within HA for the last step. It make take one to two clicks on Confirm but it has worked for two  Dyson fans for me.

**Comment by pelu89** [comment]
> I was able to get this to work by using Cloud Local. Grab the Wifi SSID and password from the device. If you removed the sticker, it is behind the filter. Put the Dyson into wifi pairing mode. Start by adding the device in HA using Dycon local and select Wifi, enter you Wifi SSID and password but dont hit confirm. Then switch over to the Dyson app on your phone and start to add fan to the Dyson app. After you have connected the phone to the Wifi of the Dyson and the app is transferring the Wifi settings, you can click confirm within HA for the last step. It make take one to two clicks on Confirm but it has worked for two Dyson fans for me.

Yeah it seems that my Dyson does not support this. There is no SSID an PW anywhere. It connects to the phone via bluetooth as far as I know and then sets it up, so this was not an option for me unfortunately.
